gef-ui/server/controllers/activities-controller/index.test.js

```diff
@@ -1,19 +1,25 @@
-import { fromUnixTime } from 'date-fns';
+import { fromUnixTime, format } from 'date-fns';
+import { timeZoneConfig, DATE_FORMATS } from '@ukef/dtfs2-common';
 import { mapPortalActivities, getPortalActivities } from '.';
 import api from '../../services/api';
 import mocks from '../mocks';
 
 jest.mock('../../services/api');
 
+const timestamp = 1733311320;
+const date = fromUnixTime(new Date(timestamp));
```

Comment:
This isn't doing what it seems...
-  `fromUnixTime` should take a timestamp not a date, its only functioning because Dates are coerced into numbers automatically
- Moreover `new Date(1733311320)` returns januray 1970, not December 2024 as you might expect

```suggestion
const timestamp = 1733311320;
const date = fromUnixTime(timestamp);
```